### PR TITLE
Update smartbch rpc with fallbacks

### PR DESCRIPTION
**Pull request by amalektricity**
Update smartbch rpc with more fallbacks

Fixes # single dependency on one RPC node which was offline, causing vote weight to not calculate properly

Changes proposed in this pull request:
- Add addditional RPC endpoints for greater fallback

### CODE REVIEW

**Comment by amalektricity** [comment]
@ChaituVR If you can please help review when you have a moment, currently all snapshot votes for this chain are not functioning.

**Comment by ChaituVR** [comment]
Hi @amalektricity Can plz share me the proposal you are facing issue with?

**Comment by amalektricity** [comment]
Hi @ChaituVR I added one more commit as I believe the issue isn't the RPC endpoint but rather the explorer. 

The old explorer that was in networks has been retired by the dev, I've updated it to a new version which is online

You can test this out here with this test proposal: 
https://snapshot.org/#/joystickclub.eth/proposal/0xc3cbc8652d804628297f7c43dea353b779cb9228c247be506d8a83c8df09fecc

It throws an error that appears to originate from https://www.smartscan.cash/block/12590684 being unreachable. 

This new PR and commit will provide more redundancy to RPC's and ensure that it can check the block using an active explorer. See  https://smartbch-explorer.web.app/block/12590684

File: src/networks.json [hunk 1]
```diff
@@ -1080,10 +1080,13 @@
     "network": "mainnet",
     "multicall": "0x1b38EBAd553f218e2962Cb1C0539Abb5d6A37774",
     "rpc": [
-      "https://smartbch.greyh.at/"
+      "https://global.uat.cash",
+      "https://smartbch.fountainhead.cash/mainnet",
+      "https://smartbch.greyh.at",
+      "https://rpc.smartbch.org"
```
Comment (ChaituVR) [inline src/networks.json:1086]:
```suggestion
      "https://smartbch.greyh.at/"
```

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**